saleor/core/tasks.py

```diff
@@ -1,8 +1,14 @@
 from django.core.files.storage import default_storage
 
 from ..celeryconf import app
+from .management.commands.delete_event_payloads import delete_event_payloads
```

Comment:
Can we extract `delete_event_payloads` somewhere?  It could be even here.
IMHO import should be in the reverse direction. The command should import logic to execute from Saleor.